GANDLF/models/brain_age.py

```diff
@@ -27,7 +28,7 @@ def brainage(parameters):
         # Load the pretrained VGG16 model
         model = torchvision.models.vgg16(pretrained=True)
     except Exception:
-        sys.exit("Error: Failed to load VGG16 model: " + traceback.format_exc())
+        logging.error(f"Error: Failed to load VGG16 model: {traceback.format_exc()}")
```

Comment:
Here as with the above replacements - I believe Exception should be still thrown

Reply:
Same response - let's wait for the next PR.